Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -1241,28 +1241,116 @@ void MainWindow::removeManipulatedFrame(CGAL::Three::Scene_item* item)
 }
 
 void MainWindow::updateInfo() {
-  CGAL::Three::Scene_item* item = scene->item(getSelectedSceneItemIndex());
-  if(item) {
-    QString item_text = item->toolTip();
-    QString item_filename = item->property("source filename").toString();
-    item_text += QString("Bounding box: min (%1,%2,%3), max(%4,%5,%6)")
-            .arg(item->bbox().xmin)
-            .arg(item->bbox().ymin)
-            .arg(item->bbox().zmin)
-            .arg(item->bbox().xmax)
-            .arg(item->bbox().ymax)
-            .arg(item->bbox().zmax);
-    if(item->getNbIsolatedvertices() > 0)
-    item_text += QString("<br />Number of isolated vertices : %1<br />").arg(item->getNbIsolatedvertices());
-    if(!item_filename.isEmpty()) {
-      item_text += QString("<br /><i>File: %1").arg(item_filename);
+  int null_edges(0), null_facet(0),isolated(0), polys(0), vertices(0), edges(0), facets(0), total_edges(0), holes(0);
+  double min_edge(std::pow(2, sizeof(double))), max_edge(0);
+  double mean(0);
+  bool all_triangle = true, none_self_interset = true;
+  Q_FOREACH(int id, getSelectedSceneItemIndices())
+  {
+    Scene_polyhedron_item* item =  qobject_cast<Scene_polyhedron_item*>(scene->item(id));
+    if(item)
+    {
+      all_triangle &= item->triangulated();
+      none_self_interset &= item->self_intersected();
+      polys++;
+      vertices += item->polyhedron()->size_of_vertices();
+      edges += item->polyhedron()->size_of_halfedges()/2;
+      facets += item->polyhedron()->size_of_facets();
+      null_edges += item->getNumberOfNullLengthEdges();
+      null_facet += item->getNumberOfDegeneratedFaces();
+      isolated += item->getNbIsolatedvertices();
+      max_edge = CGAL::max(max_edge, item->getMaxEdgesLength());
+      min_edge = CGAL::min(min_edge, item->getMinEdgesLength());
+      mean += item->getMeanEdgesLength() * item->polyhedron()->size_of_halfedges()/2;
+      total_edges += edges;
+
```

Comment:
All that code should be in the `toolTip()` function of `Scene_polyhedron_item`, and not in the main window.